src/main/java/org/jpeek/calculus/java/Ccm.java

```diff
@@ -105,4 +122,92 @@ private static void updateNcc(
+                    continue;
+                }
+                final Node child = createPackageTag(doc, packages);
+                meta.appendChild(child);
+                packages = packages.getNextSibling();
+            }
+            doc.appendChild(meta);
+        } catch (final ParserConfigurationException ex) {
+            throw new IllegalStateException(ex);
+        }
+        final Node repr = skeleton.node().getFirstChild();
+        final Node node = repr.getOwnerDocument()
+            .importNode(doc.getDocumentElement(), true);
+        repr.appendChild(node);
+        return new XMLDocument(repr);
+    }
+
+    /**
+     * Creates a package tag element in the XML document.
+     * @param doc The XML document to which the package tag will be added
+     * @param packages The node representing the package
+     * @return The package tag element created in the XML document
+     */
+    private static Node createPackageTag(final Document doc, final Node packages) {
+        final NamedNodeMap map = packages.getAttributes();
+        final Element root = doc.createElement("package");
+        final Node id = map.getNamedItem("id");
+        root.setAttribute("id", id.getNodeValue());
+        Node classes = packages.getFirstChild();
+        while (Objects.nonNull(classes)) {
+            if (classes.getTextContent().matches("\\s*")) {
+                classes = classes.getNextSibling();
+                continue;
+            }
+            final Node child = createClassTag(doc, classes);
+            root.appendChild(child);
+            classes = classes.getNextSibling();
+        }
+        return root;
+    }
+
+    /**
+     * Creates a class tag element in the XML document.
+     *
+     * This method constructs a class tag element in the XML document based on the
+     * information provided by the given node representing a class. It traverses
+     * through the attributes of the class node to extract necessary information
+     * and constructs the class tag element accordingly.
+     *
+     * Additionally, this method adds a child element representing a metric called
+     * "ncc" to the class tag.
+     *
+     * @param doc The XML document to which the class tag will be added
+     * @param classes The node representing the class
+     * @return The class tag element created in the XML document
+     * @todo #522:30m/DEV Implement method that would build graph where methods are nodes and
+     *  field references are edges and find connected components in that graph.
+     */
+    private static  Node createClassTag(final Document doc, final Node classes) {
+        final NamedNodeMap map = classes.getAttributes();
+        final Element root = doc.createElement("class");
+        final Node id = map.getNamedItem("id");
+        root.setAttribute("id", id.getNodeValue());
+        final Integer value = 1;
+        final Element ncc = doc.createElement("ncc");
```

Comment:
not only ncc metric should be presented, you can check current implementation 
https://user-images.githubusercontent.com/63350301/117303386-4ff0da00-ae85-11eb-82c3-869f1a66521d.png